deploy.py

```diff
@@ -23,7 +25,8 @@ def run():
     iam = boto3.client("iam")
     # delete the role if it already exists, so it can be deployed with
     # the latest configuration
-    roles = iam.list_roles()["Roles"]
+    
+    roles = iam.list_roles(MaxItems=1000)["Roles"]
```

Comment:
I would prefer if this was actually paginated in case we go over 1000. Example here: https://blog.ruanbekker.com/blog/2019/01/29/paginate-through-iam-users-on-aws-using-python-and-boto3/

Reply:
done